wordpress-mobile/WordPress-iOS
# Issue/fix url selection bug

This PR works around what it looks to be a bug on iOS 13 around link interactions in UITextView.

According to the [documentation](https://developer.apple.com/documentation/uikit/uitextviewdelegate/1649337-textview).

```
Links in text views are interactive only if the text view is selectable but noneditable. 
That is, if the value of the UITextView isSelectable property is true and the isEditable property is false.
```

Aztec textview has the isSelectable property to true but we are getting interactive links anyway.

So this PR works around it by not allowing interaction and then selecting the initial position of the URL range.

To test:
 - Open or create a post with a link on the content using the Aztec editor
 - Tap on the link
 - Make sure that Safari is not open to the page of the tapped link.

Update release notes:
